pkgs/by-name/ni/nixedit/package.nix

```diff
@@ -0,0 +1,73 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  bash,
+  fzf,
+  jq,
+  micro,
+  git,
+  nix-tree,
+  coreutils,
+  makeWrapper,
+  dialog,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nixedit";
+  version = "1.0.0";
```

Comment:
Please read https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning.

Reply:
Can you be more specific? 1.0.0 is the correct version.

Reply:
It's not since upstream doesn't have such a version.